Sources/Core/Int32.swift

```diff
@@ -6,6 +6,21 @@
 public struct Int32 {
   @usableFromInline
   internal var _value: Builtin.Int32
+
+  @_transparent
+  public init(_ _value: Builtin.Int32) {
+    self._value = _value
+  }
+
+  @_transparent
+  public static func &= (lhs: inout Int32, rhs: Int32) {
```

Comment:
Stylistic note: these parameters are not labeled when passed, so it is nicer to use `_ ...` here IMO.